libmamba/include/mamba/core/solver.hpp

```diff
@@ -25,9 +25,9 @@
 
 #include "match_spec.hpp"
 
-#define MAMBA_NO_DEPS 0b0001
-#define MAMBA_ONLY_DEPS 0b0010
-#define MAMBA_FORCE_REINSTALL 0b0100
+#define PY_MAMBA_NO_DEPS 0b0001
```

Comment:
I believe these can be replaced by an enum or at least constexpr int.

Reply:
These are kept for 1.x compatibility but will be removed.